frog/imports/client/Wiki/index.js

```diff
@@ -52,13 +52,19 @@ export const LearningItem = dataFn.LearningItem;
 
 type WikiCompPropsT = {
   location: *,
-  match: {
+  match?: {
```

Comment:
If we are making this field optional, what happens if functions that use this prop receive `undefined`?

Reply:
The only case that this could happen is when it is not called by the `react-router` (which is what happens when a user goes to the URL). This would imply that the Wiki is being embedded.

Reply:
What would be the expected behaviour in this case? Would enabling a flag (`isWikiEmbeded` or similar) work better?